We are not interested in tests suitable primarily for setuptools authors and
maintainers.

--- setuptools-80.10.1/tox.ini.orig
+++ setuptools-80.10.1/tox.ini
@@ -10,10 +10,7 @@
 usedevelop = True
 extras =
 	test
-	check
-	cover
 	enabler
-	type
 	core
 pass_env =
 	SETUPTOOLS_USE_DISTUTILS
